src/NuProj.Package.Dev14/NuProj.Package.Dev14.csproj

```diff
@@ -17,7 +17,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NuProj.ProjectSystem</RootNamespace>
-    <AssemblyName>NuProj.ProjectSystem</AssemblyName>
+    <AssemblyName>NuProj.ProjectSystem14</AssemblyName>
```

Comment:
Looking at the references, it's either `.V14Only` for project system, `.v12.0` msbuild, or `.12.0` for almost everything else. I'd probably go for the last option.